### PR TITLE
FUSETOOLS-3074 - include Fuse 7.1

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [x] Did the CI job report a successful build?
- [x] Is the non-happy path working, too?

## Maintainability

- [x] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [x] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+							<goal>copy</goal>
+						</goals>
+						<phase>generate-resources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<artifactItems>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-catalog</artifactId>
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: indention

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+							<artifactId>camel-catalog-provider-springboot</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-core</artifactId>
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: indention

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-spring</artifactId>
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: indention

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-blueprint</artifactId>
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: indention

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+				        <artifactItem>
```
Comment (lhein) [inline core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: indention

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml (hunk 1 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: fixed, don't know why Github still showing the comment

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml (hunk 2 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: fixed, don't know why Github still showing the comment

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml (hunk 3 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: fixed, don't know why Github still showing the comment

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml (hunk 4 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: fixed, don't know why Github still showing the comment

File: core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml (hunk 5 above)
Comment (apupier) [reply core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml]: fixed, don't know why Github still showing the comment

**Pull request merged**